### ISSUE TITLE
Package 'libxtrxdsp' and 'libxtrxll' not found

### ISSUE THREAD

**Issue by teon** [xtrx-sdr/libxtrx#8]
I have compiled (cmake . && make && sudo make install) libxtrxll and libxtrxdsp - and when I try to do the same wit libxtrx I get:

```
-- Checking for module 'libxtrxdsp'
--   Package 'libxtrxdsp', required by 'virtual:world', not found
-- libxtrxdsp not found.
-- Checking for module 'libxtrxll'
--   Package 'libxtrxll', required by 'virtual:world', not found
-- libxtrxll not found.
```

I can't even test my XTRX ;-( - generally would be a good idea to have a WIKI.. whatever how to install & test xtrx... Hardware without this software is unusable...

**Comment by chemeris** [comment]
Build instructions are available at https://github.com/xtrx-sdr/images
Let us know if this helps.

We plan to create a proper wiki but haven't done this yet, sorry.

**Comment by teon** [comment]
@chemeris figured that out after a while - but it did not help. Can't get my XTRX running since the sources do not compile:

https://github.com/xtrx-sdr/images/issues/46

**Comment by earwin** [comment]
Resolved in xtrx-sdr/images#46

**Comment by markl17** [comment]
do I have build it or can I just apt-get it
